ixofoundation/ixo-webclient
# feat(bot): fuel project skeleton

## Scope

https://nonaredteam.atlassian.net/browse/IXO-558

## Work Done

Actions will open bot with an intent
Developed skeleton for fueling a project

## Steps to Test

Make sure you have this in your env file
REACT_APP_ASSISTANT_URL=https://assistant.ixo.world
Click on Fuel Project from the Control Panel
Bot will appear
Enter any message and see that summary screen is shown with mock values

## Screenshots

![image](https://user-images.githubusercontent.com/20910282/83000374-8cc41800-a00a-11ea-8b4a-0a617e7ea144.png)

![image](https://user-images.githubusercontent.com/20910282/83000401-98afda00-a00a-11ea-8c23-0abd8a1e3a10.png)
